code/__DEFINES/preferences_defines.dm

```diff
@@ -77,7 +77,21 @@
 #error toggles_2 bitflag over 16777215. Please make an issue report and postpone the feature you are working on.
 #endif
 
-
+// Preftoggle defines
+#define PREFTOGGLE_GENERAL		1
```

Comment:
Not gonna get too indepth with a review, but you should probably preface some of these defines, something like category may be good
```suggestion
#define PREFTOGGLE_CATEGORY_GENERAL		1
```